RestSharpClient/ApiClient/ApiService.cs

```diff
@@ -0,0 +1,242 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using Newtonsoft.Json.Converters;
+using System.Globalization;
+using System.Reflection.PortableExecutable;
+using System.Net.Mime;
+using RestSharp.Authenticators;
+using RestSharpClient.Model.RestfulBooker;
+using Settings;
+using RestSharpClient.Utils;
+using RestSharp.Serializers;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using System.Runtime.CompilerServices;
+using Helpers;
+using JsonSerializer = RestSharpClient.Utils.JsonSerializer;
+
+namespace RestSharpClient.ApiClient
+{
+    public partial class ApiService : BaseClient, IApiService, IDisposable
```

Comment:
IDisposable - probably better inherited to IApiService

Reply:
Makes better sense. Moved that and later after realizing the IDisposable is already implemented with RestSharp and since I dont have any specific application for that, called off inheriting IDisposable. Thanks!